cli/internal/mcp/instructions-plugin-library-development.md

```
@@ -453,6 +455,112 @@ terraform {
+### Step 1: Start the Plugin Development Server
+
+From your plugin library root directory:
+
+```bash
+cd my-plugins
+suga plugin serve
+```
+
+You'll see output like:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)